src/libs/actions/BankAccounts.js

```diff
@@ -906,6 +906,22 @@ function validateRoutingNumber(number) {
     return false;
 }
 
+function getOAuthReceivedRedirectURI() {
+    if (!window.location) {
+        return null;
+    }
+
+    let receivedRedirectURI = window.location.href;
+    const receivedRedirectSearchParams = (new URL(receivedRedirectURI)).searchParams;
+    const oauthStateID = receivedRedirectSearchParams.get('oauth_state_id');
+
+    // If there's no stateID passed, then return user to start of Plaid flow by setting the redirectURI to null
+    if (!oauthStateID) {
+        receivedRedirectURI = null;
```

Comment:
NAB, it would be slightly clearer IMO to just return `null` here instead of reassignment.